projects/packages/backup/src/class-rest-controller.php

```diff
@@ -616,6 +628,66 @@ public static function get_site_backup_undo_event() {
 		return rest_ensure_response( $undo_event );
 	}
 
+	/**
+	 * Fetch a backup of a order, along with all of its data.
+	 *
+	 * @access public
+	 * @static
+	 *
+	 * @param WP_REST_Request $request The request sent to the WP REST API.
+	 *
+	 * @return array
+	 */
+	public static function fetch_wc_orders_backup( $request ) {
+		global $wpdb;
+
+		// Disable Sync as this is a read-only operation and triggered by sync activity.
+		Sync_Actions::mark_sync_read_only();
+
+		$order_id = $request['id'];
+
+		$order                  = array();
+		$order_addresses        = array();
+		$order_operational_data = array();
+		$order_meta             = array();
+
+		if ( ! class_exists( 'Automattic\WooCommerce\Internal\DataStores\Orders\OrdersTableDataStore' ) ) {
+			return new WP_Error( 'order_not_allowed', __( 'Not allowed to get the order with current configuration', 'jetpack-backup-pkg' ), array( 'status' => 403 ) );
+		}
+
+		if ( method_exists( 'Automattic\WooCommerce\Internal\DataStores\Orders\OrdersTableDataStore', 'get_orders_table_name' ) ) {
+			// phpcs:ignore WordPress.DB.DirectDatabaseQuery.DirectQuery,WordPress.DB.DirectDatabaseQuery.NoCaching,WordPress.DB.PreparedSQL.NotPrepared
+			$order = $wpdb->get_row( $wpdb->prepare( 'SELECT * FROM `' . OrdersTableDataStore::get_orders_table_name() . '` WHERE id = %s', $order_id ) );
```

Comment:
Using `get_orders_table_name()` to get the `wc_orders` table name if the plugin exists.